kebab/package-lock.json

```diff
@@ -0,0 +1,5716 @@
+{
```

Comment:
please don't use npm on kebab, we use yarn for almost all of our workspaces. also delete the `package-lock.json`